monocle/bounds.py

```diff
@@ -38,12 +38,21 @@ def __init__(self, polygon=conf.BOUNDARIES):
         self.south, self.west, self.north, self.east = polygon.bounds
         self.center = polygon.centroid.coords[0]
         self.multi = False
+        self.prepared = False
```

Comment:
How about instead of having and checking this Boolean we just give it a name that will always be present?

I'm thinking we should rename `self.boundaries` to `self.polygon` and `self.prepared_boundaries` to `self.boundaries`.